lambda/permissionUtil.js

```diff
@@ -132,15 +144,6 @@ exports.roleFilter = function (records, roles) {
 };
 
 exports.resolvePermissions = function (token) {
-
-  if (process.env.IS_OFFLINE == true) {
-    return {
-      roles: ['sysadmin'],
-      isAdmin: true,
-      isAuthenticated: true
-    }
-  }
```

Comment:
This would need to be put back in based on my comment above.

Reply:
This section restored as per above, with a check for it not to be in `process.env.IS_TEST`.